pkg/release/unpacker.go

```diff
@@ -126,6 +134,39 @@ func unpackUBIImagesFromRelease(ctx context.Context, s *Staged) (map[string][]im
 	return unpackImages(ctx, ubiA, "-ubi")
 }
 
+// unpackCtlBinariesFromRelease will extract all ctl binary tar archives
+// from the various 'ctl' .tar.gz files and return a map of component name
+// to a slice of binaries.Tar for each image in the bundle.
+func unpackCtlBinariesFromRelease(ctx context.Context, s *Staged) (map[string][]binaries.Tar, error) {
+	log.Printf("Unpacking 'ctl' type artifacts")
+	ctlA := s.ArtifactsOfKind("ctl")
+
+	// tarBundles is a map from component name to slices of images.Tar
+	tarBundles := make(map[string][]binaries.Tar)
+	for _, a := range ctlA {
+		dir, err := extractStagedArtifactToTempDir(ctx, &a)
+		if err != nil {
+			return nil, err
+		}
+		imageArchives, err := recursiveFindWithName(dir, "cert-manager-ctl")
+		if err != nil {
+			return nil, err
+		}
+		for _, archive := range imageArchives {
+			imageTar, err := binaries.NewTar(archive, a.Metadata.OS, a.Metadata.Architecture)
```

Comment:
`binaries.NewTar`? 🤔

Reply:
comes from a modified images/tar.go

Reply:
fixed this, got confused with Tar files being used everywhere